### PR TITLE
Add ruby getting started

### LINKED ISSUE #238: Missing Getting Started with Ruby

### PR DESCRIPTION
**Pull request by johnsudaar**
Fix #238

### CODE REVIEW

File: _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+modified_at: 2018-02-23 00:00:00
+category: getting-started
+tags: ruby sinatra tutorial getting-started-tutorial
+index: 3
+permalink: /languages/ruby/getting-started-with-ruby/
+---
+
+## Installing the tools
+
+This tutorial will show you how to deploy a ruby app in minutes.
```
Comment (Soulou) [inline _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md]: Why 'you' ?

Comment (Soulou) [reply _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md]:
```
This tutorial is depicting how to deploy a ruby application on Scalingo
```

File: _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+Note: This tutorial does not cover the rails installation, if you want to start a rails project use our [getting started with rails tutorial]({% post_url languages/ruby/2015-02-17-getting-started-with-rails %})
```
Comment (Soulou) [inline _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md]: Note should be in `<aside class="note" markdown="1"></aside>`

Comment (Soulou) [reply _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md]: `the rails installation` -> `Ruby on Rails installation`

Comment (Soulou) [reply _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md]:
```
if you want to start a rails project
```
-> 

```
If you're looking to deploy a Ruby on Rails project, follow the [getting started ...
```
```

**Comment by Soulou** [comment]
A few remarks:

- Limit the usage of `you` and `your`
- Do not use `we` or `our`, use the passive form:

```
We expect you to have
->
You are expected to have
```

**Comment by johnsudaar** [comment]
Why making this always so formal? 
Look at the Heroku doc, they talk to YOU the reader all the time and I like it.

**Comment by johnsudaar** [comment]
Even AWS use YOU all the time https://docs.aws.amazon.com/AmazonS3/latest/dev/website-hosting-custom-domain-walkthrough.html

**Comment by Soulou** [comment]
I dont' say you have to remove the `you`, just to limit it, in order to avoid sentences like `you should create this file in your application directory` -> `This file should be created in your application directory`

**Comment by Soulou** [comment]
Or even, using the imperative form `Create this file in your application directory`

File: _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+```bash
+$ bundle init
+```
+
+This will create a file name `Gemfile`. This file will be used for describing gem dependencies for your ruby application.
+
+
+## Create your ruby application
+
+<blockquote class="bg-info">
```
Comment (yannski) [inline _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md]: no fucking blockquote

Comment (johnsudaar) [reply _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md]: What should i use instead.
Specify it in the fucking README

Comment (Soulou) [reply _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md]: Note should be in `<aside class="note" markdown="1"></aside>`

Comment (Soulou) [reply _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md]: So far it's present here:  https://github.com/Scalingo/homepage (README), I'd like @yannski to create some doc on `scalingo-styles` if possible to let us know this kind of things

Comment (johnsudaar) [reply _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md]: Right ...

2018-02-23_143x54_scrot

2018-02-23_395x69_scrot

Comment (Soulou) [reply _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md]: You syntax highlighter is not great =P

Comment (Soulou) [reply _posts/languages/ruby/2018-02-23-getting-started-with-ruby.md]: Think of the future @johnsudaar , the new template is coming

**Pull request merged**